tests/x11regressions/empathy/empathy_irc.pm

```diff
@@ -48,16 +48,13 @@ sub run() {
     };
 
     # join openqa channel
-    if (sle_version_at_least('12-SP2')) {
+    send_key "ctrl-j";
+    if (!check_screen 'empathy-join-room', 4) {
```

Comment:
My question is more about whether this is the right way to go.

I personally see a race condition in this firefox.pm example. Isn't it possible that 'firefox-save-and-quit' will match before firefox quit??? However, in this case it's harmless because sending 'ret' to an empty desktop will do nothing. What will happen though if you have assert_screen inside that if statement?